apply_patches.sh

```diff
@@ -3,24 +3,35 @@
 set -e
 
 if [[ $# < 1 ]]; then
-    echo "apply_patches.sh [--one-cam] apply [JetPack_version]"
+    echo "apply_patches.sh [--one-cam | --dual-cam] apply [JetPack_version]"
     echo "apply_patches.sh reset [JetPack_version]"
     exit 1
 fi
 
-# Default to dual camera DT for JetPack 5.0.2
-JP5_D4XX_DTSI="tegra194-camera-d4xx.dtsi"
+# Default to single camera DT for JetPack 5.0.2
+# single - dev board
+# one/dual - evb
+JP5_D4XX_DTSI="tegra194-camera-d4xx-dual.dtsi"
```

Comment:
The comment on line 11 says default to single camera, but on line 14 it looks like the dual dtsi is the actual default?

Reply:
ok done.